tests/scripts/thread-cert/Cert_7_1_06_BorderRouterAsLeader.py

```diff
@@ -327,13 +326,14 @@ def verify(self, pv):
                               SOURCE_ADDRESS_TLV,
                               LEADER_DATA_TLV,
                              } <= set(p.mle.tlv.type) and\
-                   [Ipv6Addr(PREFIX_2001[:-3])] ==
-                   p.thread_nwd.tlv.prefix and\
-                   [ROUTER_2_RLOC16] == p.thread_nwd.tlv.border_router_16 and\
                    p.mle.tlv.leader_data.data_version ==
                    (_dr_pkt1.mle.tlv.leader_data.data_version + 1) % 256 and\
                    p.mle.tlv.leader_data.stable_data_version ==
-                   (_dr_pkt1.mle.tlv.leader_data.stable_data_version + 1) % 256
+                   (_dr_pkt1.mle.tlv.leader_data.stable_data_version + 1) % 256 and\
+                   is_sublist([Ipv6Addr(PREFIX_2001[:-3])],
+                              p.thread_nwd.tlv.prefix) and\
+                   is_sublist([ROUTER_2_RLOC16],
+                              p.thread_nwd.tlv.border_router_16)
```

Comment:
To check `Router_1’s Network Data section MUST be removed`, should we check ROUTER1's short address is not in the list?